Entity/Reason.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of aakb/kontrolgruppen-core-bundle.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace Kontrolgruppen\CoreBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="Kontrolgruppen\CoreBundle\Repository\ReasonRepository")
+ * @UniqueEntity("name")
```

Comment:
Perhaps this should be added to AbstractTaxonomy instead, since it should apply to all taxonomies?